client/src/ui/organisms/placement/index.tsx

```diff
@@ -264,21 +283,17 @@ export function PlacementInner({
         timer.current.timeout === null
       ) {
         timer.current = {
-          timeout: window?.setTimeout?.(
-            () => viewed(pong, observer?.current),
```

Comment:
this should have been the same as [above](#diff-835d1b59f9edf5292053eb91c51b31ee54a67df4e8ce67713b04d4bfe5d05ec5L224)